packages/core/ios/LottieReactNative/ContainerView.swift

```diff
@@ -296,8 +289,6 @@ class ContainerView: RCTView {
     func replaceAnimationView(next: LottieAnimationView) {
         super.removeReactSubview(animationView)
 
-        let contentMode = animationView?.contentMode ?? .scaleAspectFit
-
         animationView = next
 
         animationView?.contentMode = contentMode
```

Comment:
If the functionality is being extracted, i suppose this no longer makes any sense to be here